library/src/main/java/com/fraggjkee/smsconfirmationview/SmsConfirmationView.kt

```diff
@@ -202,7 +279,8 @@ class SmsConfirmationView @JvmOverloads constructor(
 
     @SuppressLint("ClickableViewAccessibility")
     override fun onTouchEvent(event: MotionEvent): Boolean {
-        if (event.action == MotionEvent.ACTION_DOWN && requestFocus()) {
+        if (event.action == MotionEvent.ACTION_DOWN && !this.isFocused) {
```

Comment:
Looks like this change breaks the keyboard: open demo app -> dismiss keyboard -> try opening it again by clicking on the view-> woops...